contracts/governance/airdrop.cairo

```diff
@@ -0,0 +1,2463 @@
+%lang starknet
+
+@storage_var
+func airdrop_claimed(claimee: Address) -> (res: felt) {
+}
+
+@external
+func claim{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(claimee: Address) {
+    alloc_locals;
+
+    let (governance_token_addr) = governance_token_address.read();
+
+    let claimed_already = airdrop_claimed.read(claimee);
```

Comment:
so, going with having "claimee" as input variable means that anyone can claim tokens for anyone?

instead going with get_caller_address

Reply:
Yes

You can basically pay for anyone's tokens being sent to their 

One issue I see that we're calling totalSupply in check_proposal_passed_express and that doesn't change until tokens are actually claimed